### DISCUSSION TITLE
Printing Time

### DISCUSSION THREAD

**Discussion by JALTAV** [mriscoc/Ender3V2S1#861] [Q&A]
Hi,

When a print job is too long i turn LCD off, then when i want to see the remaining time i have to go back to main menu.
Is it possible to display the remaining time  in the status line whem i turn on the LCD
Thanks

**Comment by mriscoc** [comment]
Hi, the remaining time is there:
![image](https://user-images.githubusercontent.com/2745567/232259639-816228e8-dd8b-4d1f-82c6-a3bbea939f26.png)

**Reply by mriscoc** [reply]
You can send text to the status line using a plug-in as the `Display Progress on LCD` for Cura and the M117 mode.

**Reply by JALTAV** [reply]
ok, thanks, i will try the plug-in

**Comment by TheWebMachine** [comment]
I think I get what he is asking but I think the solution would be to have the LCD Off menu item return to main print status screen after turning off the LCD. As it is currently coded, when you press the encoder wheel to wake the LCD, it is still deep in the menus with LCD Off selected and you have to back out to see the current status. If LCD Off turned off LCD *and* returned to main status screen, it would be faster to see status when LCD comes back on.

**Reply by mriscoc** [reply]
I think that you are right

**Reply by JALTAV** [reply]
Yes, but if the remaining time is too long, i want to turn off LCD, not a big deal.
Congrats for your great work

**Comment by JALTAV** [comment]
Thank you Miguel, **Display Progress on LCD** works great.
